.lighthouserc.yml

```diff
@@ -0,0 +1,17 @@
+ci:
+  assert:
+    assertions:
+      categories:accessibility: [error, minScore: 0.94]
+      categories:best-practices: [error, minScore: 1]
+      categories:performance: [error, minScore: 0.68]
+      categories:seo: [error, minScore: 0.92]
+    includePassedAssertions: true
+  collect:
+    numberOfRuns: 1
+    startServerCommand: http-server dist-ssr --brotli -c31536000
+    startServerReadyPattern: Hit CTRL-C to stop the server
+    url:
+      - http://localhost:8080/en
+  upload:
+    outputDir: .lighthouseci/results
+    target: filesystem
```

Comment:
No, they aren't, but the upload must go somewhere, so I just put them in an ignored-anyway folder. If you are fine with different values feel free to suggest some:
https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#target

Reply:
Thanks for the investigation.

`filesystem` is ok, but how about moving it to `tmp/lighthouseci/results` and gitignore `tmp`?

Reply:
`.lighthouseci` is written to anyway (see https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#collect).

What we could do is something like:
```yaml
  upload:
    outputDir: .lighthouseci
    reportFilenamePattern: tmp
    target: filesystem
```
This would no longer accumulate reports for local runs and even hide the (over)written dummy file from most IDEs.
But in case someone is actually interested in archived local results, this would be a blocker. Leave as is?